install crd with the chart (#552)
giantswarm/silence-operator: CHANGELOG.md

## [Unreleased]

### Added

> - Helm chart now supports conditional installation of the Silence CRD via the `crds.install` value. The CRD is templated and will only be installed if `crds.install` is set to `true`.

Reviewer: ```suggestion
- Helm chart now supports conditional installation of the Silence CRD via the `crds.install` value. The CRD is templated and installed by default, but you can disable it by setting `crds.install` to `false`.
```
